src/ui/screens/CreateAccount/CreateAccountScreen.test.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {NavigationProp, RouteProp} from '@react-navigation/native';
+import {AccountsStackParamList} from '@ui/navigation/navigation';
+import {render, fireEvent} from 'src/testUtils';
+import {CreateAccountScreen} from './CreateAccountScreen';
+import {createAccountScreen} from '@ui/navigation/routeKeys';
+import {useKeyring} from '@polkadotApi/useKeyring';
+
+const navigation = {
+  navigate: () => jest.fn(),
+  goBack: () => jest.fn,
+} as unknown as NavigationProp<AccountsStackParamList>;
+
+const route = {
+  params: {
+    mnemonic: 'west bar upon arena all remove return era local spoon edge use',
+  },
+} as RouteProp<AccountsStackParamList, typeof createAccountScreen>;
+
+const mockCreateAddressFromMnemonic = jest.fn(() => Promise.resolve());
```

Comment:
This mock should return an address and if we display it in the screen you should assert that is displayed